web/controllers/user_controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule MovieWagerApi.UserController do
+  use MovieWagerApi.Web, :controller
+
+  alias MovieWagerApi.{Repo, User, UserSerializer}
+
+  def show(conn, %{"id" => identifier}) do
```

Comment:
discussed IRL and agreed that it's not ideal to adjust the user table's primary key to accommodate Ember's propensity to create zombie records :)